### DISCUSSION TITLE
Can saga is used in distribution？

### DISCUSSION THREAD

**Discussion by marjezhang** [rasmus/EventFlow#869] [Q&A]
My program have 3 aggregate like order、payment、reserve，they are distributed in different application like microservice.
Their eventhandl in the saga aggregate are not triggered in this time。
How can i use the saga in distribution system? 
Is it fault like up doing, or it's fault in my mind about saga or eventbus?

**Comment by MongkonEiadon** [comment]
I've not tried this case yet, but still on my head,

The Saga is started by specific event so that mean the Saga must be implemented "ISagaIsStartedBy<>", for my opinion every microservice should have the same "EventBus" so the saga can know when the Event rised ...
